src/screens/UserManagement/UserRevamp/NewUserInvitationForm.res

```diff
@@ -186,9 +186,11 @@ let make = () => {
         />
       </div>
     </div>
-    <div className="grid grid-cols-5 h-full">
+    <div className="grid md:grid-cols-5 grid-cols-2 h-full">
       <div className="col-span-2 border-r p-6  flex flex-col gap-2">
-        <OrganisationSelection />
+        <div className="overflow-scroll">
```

Comment:
https://github.com/user-attachments/assets/66a0dfb1-9c25-4702-a512-3eab3c9a79a6

I think you can do something else to fix this on larger screen size

Reply:
Made the changes without overflowing